ui/src/shared/components/TimeMachine/FluxQueryMaker.tsx

```diff
@@ -258,8 +258,10 @@ class FluxQueryMaker extends PureComponent<Props, State> {
   }
 
   private handleAppendFrom = (): void => {
+    const {source} = this.props
     const {draftScript} = this.state
-    const from = builder.NEW_FROM
+
+    const from = builder.getNewFromScript(source.telegraf, source.defaultRP)
```

Comment:
should there be a fallback if the `telegraf` database doesn't exist?

Reply:
i believe there will always be a telegraf. even if you update your source and get rid of the telegraf db name, it is added back in and defaults to `telegraf`